### PR TITLE
[APM-898] Run e2e tests using Angular CLI

### PR DESCRIPTION
**Pull request by rdiacenco**
* Use 'npm run e2e' in travis build script
* Remove test-e2e-ama.sh script
* Update README file

**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe: E2e tests execution using Angular CLI

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "pack:sdk": "cd dist/ama-sdk && npm pack",
     "package:sdk": "npm run build:sdk && npm run pack:sdk",
     "06": "echo ------------------------------------------- E2E tests ----------------------------------------------",
-    "pree2e": "npm run compile-less",
+    "pree2e": "npm run compile-less && webdriver-manager update --gecko=false",
```
Comment (popovicsandras) [inline package.json]: No need to invocate the webdriver-manager update script here. Please check the angular cli options for e2e. There is an option for it.

Comment (rdiacenco) [reply package.json]: Indeed the update is covered by default in ng e2e. Thank you.

File: .travis.yml [hunk 2]
```diff
@@ -23,9 +23,8 @@ env:
 jobs:
     include:
         - stage: E2E Test
-          script:
-            ./scripts/test-e2e-ama.sh -host "localhost" -port 4100 -u "$E2E_USERNAME" -p "$E2E_PASSWORD" --unauthorized-user "$E2E_UNAUTHORIZED_USER" --unauthorized-user-password "$E2E_UNAUTHORIZED_USER_PASSWORD" -save -sh-url "$SCREENSHOT_URL" -sh-user "$SCREENSHOT_USERNAME" -sh-pass "$SCREENSHOT_PASSWORD" -dev
-
+          script: npm run e2e
```
Comment (popovicsandras) [inline .travis.yml]: Package.json was modified, but the webdriver update doesn't happen this way. Please see the angular.json and this page:
https://angular.io/cli/e2e
`webdriverUpdate` has to be true in angular.json (or --webdriverUpdate=true from command line) for webdriver updating happen automatically.

Comment (rdiacenco) [reply .travis.yml]: Default option for --webdriverUpdate is True, what I understood is that the update happen automatically, we don't need to specify this option.
In Travis, the build execution shows the chrome-driver update in the logs.

Comment (popovicsandras) [reply .travis.yml]: Sorry, my bad, you are right!

**Pull request merged**